### PR TITLE
Learn more instructions for setting up name

### PR DESCRIPTION
**Pull request by jefflau**

### CODE REVIEW

**Comment by makoto** [comment]
Minor point but could be great if you can fix these warnings, too.

```
Compiled with warnings.

./src/components/Error/Errors.js
  Line 4:   'WarningImage' is defined but never used      no-unused-vars
  Line 18:  'Warning' is assigned a value but never used  no-unused-vars
  Line 22:  'H2' is assigned a value but never used       no-unused-vars
  Line 30:  'Message' is assigned a value but never used  no-unused-vars

```

File: src/components/Icons/BookPen.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+            <g>
+                <rect id="Rectangle-2" fill="#4B84FF" x="0" y="0" width="16" height="20" rx="3"></rect>
+                <path d="M20,15.8125 C20,15.928875 20.0525,16.04 20.1465,16.1218125 L22.1465,17.8718125 C22.244,17.957125 22.372,18 22.5,18 C22.628,18 22.756,17.957125 22.8535,17.8718125 L24.8535,16.1218125 C24.9475,16.04 25,15.928875 25,15.8125 L25,4 L20,4 L20,15.8125 Z" id="Fill-4" fill="#4B84FF"></path>
+                <path d="M22,0 L23,0 C24.1045695,-2.02906125e-16 25,0.8954305 25,2 L25,3 L20,3 L20,2 C20,0.8954305 20.8954305,2.02906125e-16 22,0 Z" id="Rectangle-6" fill="#4B84FF"></path>
+                <rect id="Rectangle-3" fill="#FFFFFF" transform="translate(6.000000, 5.000000) scale(1, -1) translate(-6.000000, -5.000000) " x="2" y="4" width="8" height="2" rx="1"></rect>
+                <rect id="Rectangle-3-Copy" fill="#FFFFFF" transform="translate(6.000000, 9.000000) scale(1, -1) translate(-6.000000, -9.000000) " x="2" y="8" width="8" height="2" rx="1"></rect>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (makoto) [inline src/components/Icons/BookPen.svg]: screenshot 2019-02-14 at 05 35 43

Can you get rid of the end of the line thing (I think you have to add new line below)

**Comment by makoto** [comment]
Can you describe what it does and how it looks like? I can see some instruction on name page but not sure if that how it supposed to look. I suggest you deploy it to somewhere and ask Becca to review.

Also, the mobile view does not look nice. Probably better to change from three columns to one column on mobile view?

screenshot 2019-02-14 at 05 50 16

**Comment by jefflau** [comment]
Yeah that's true, mobile is not designed yet so I didn't do anything for it. I can change it to one column for now and change it if the design changes

**Comment by rliebert90** [comment]
@makoto the learn more section should look like this on desktop: 
screen shot 2019-02-18 at 1 11 10 pm

I haven't gotten to designing mobile yet, but for this section, let's have the text in one column / stacked on top of each other. Like this: 
screen shot 2019-02-18 at 1 19 00 pm

**Comment by jefflau** [comment]
I've added the collapse to one column for mobile. Deployed here: http://learn-more-ens.surge.sh

@rliebert90 if you go to your domain that you own it should show up

**Pull request merged**